scripts/generate-git-snapshot

```diff
@@ -234,7 +234,7 @@ identify_latest_change() {
   fi
 
   echo "Latest tag [${last_tag:-}] / merge [${last_merge:-}] seems to be $since"
-  ${GBP_DCH} -s "${since}" $DCH_COMBINED_OPTS
+  ${GBP_DCH} -s "${since}" $DCH_COMBINED_OPTS $DCH_CHANGELOG_FILE
```

Comment:
It's unclear to me what you're setting to DCH_CHANGELOG_FILE to, can you please provide an example with the git commit message of the PR?

Also it needs to be ${DCH_CHANGELOG_FILE:-}, otherwise the script will abort because of unset variable if DCH_CHANGELOG_FILE isn't set. :)